test/util/testnode/config.go

```diff
@@ -131,9 +132,9 @@ func DefaultConfig() *Config {
 				WithConsensusParams(DefaultConsensusParams()),
 		).
 		WithTendermintConfig(DefaultTendermintConfig()).
+		WithAppCreator(DefaultAppCreator()).
```

Comment:
👍 feel like this was a bug in that it was not being directly set in the default anyway.